pype/plugins/nuke/publish/collect_slate_node.py

```diff
@@ -8,7 +8,7 @@ class CollectSlate(pyblish.api.InstancePlugin):
     order = pyblish.api.CollectorOrder + 0.09
     label = "Collect Slate Node"
     hosts = ["nuke"]
-    families = ["write"]
+    families = ["render", "render.local", "render.farm"]
```

Comment:
here on the other hand. `render doesn't need to be there does it? each render is bound to also have `.local` or `.farm` variant of the family on it.

Reply:
In case of use only `write` it will be working on `render` and `prerender` and `.local` and `.farm` variants.
If only `render` is used it will not be active on `.local` and `.farm`

The only found solution was to set this explicitly to all three variants of render family.

Reply:
I get that. that's why I said that `render` doesn't need to be there. Only `render.local` and `render.farm`, even though as i think of it now, if there frames are already rendered, it will only have `render` family, so yes. this is correct.

Reply:
if the `render` family is missing it will not collect or create slate and that is what we don't want ir is it?